### DISCUSSION TITLE
Support skeleton generation of mill project (via giter8 integration)

### DISCUSSION THREAD

**Discussion by lilac** [com-lihaoyi/mill#1184] [Ideas]
We want to make it easy for user to get start using mill in a new project. But under current settings, we have to do a lot of manual work to get it work.

Sbt has a new command, that let us to generate a new project quickly. This is via an integration of giter8. We can do that as well.

**Comment by lefou** [comment]
Yeah, not the first time someone asks for it and it could make an easier start for newcomers as well as in corporate environments. Go ahead! I'd prefer to start with an external module but you could also target it as built-in command.

**Comment by jodersky** [comment]
FWIW, I've had success with this super simple script: https://gist.github.com/jodersky/26793057c4d0bb67f9113f0a28f56477. It's not configurable, but it contains everything to initialize a directory into a working mill project.
